build/Build.Pack.cs

```diff
@@ -145,18 +135,10 @@ void PackWindowsInstallers(MSBuildTargetPlatform platform, AbsolutePath wixNuget
                 FileSystemTasks.CopyFileToDirectory(RootDirectory / "scripts" / "Harden-InstallationDirectory.ps1", installerDirectory, FileExistsPolicy.Overwrite);
 
                 var harvestFile = RootDirectory / "installer" / "Octopus.Tentacle.Installer" / "Tentacle.Generated.wxs";
-                var restoreHarvestFileAction = RestoreFileForCleanup(harvestFile);
+                using var restoreHarvestFileAction = new ModifiedFile(harvestFile);
```

Comment:
The usage here feels a bit weird, in comparison to the `chocolateyInstallScriptFile` one above.
I wonder if an explicit block here would make it feel nicer? Or if there's a nicer name that would help?